packages/web-react/src/types/splitButton.ts

```diff
@@ -9,3 +15,25 @@ export interface SpiritSplitButtonProps<C = void, S = void> extends SplitButtonP
   color?: SplitButtonColorType<C>;
   size?: ButtonSize<S>;
 }
+
+export type UncontrolledSplitButtonProps<C = void, S = void> = {
+  buttonLabel: string;
+  buttonOnClick: () => void;
+  dropdownIconName?: string;
+  dropdownLabel?: string;
+  dropdownPlacement?: PlacementDictionaryType;
```

Comment:
maybe it would be a good idea to define some DropdownPlacements type in the dropdown.ts and export it and use it here.